app/views/devise/registrations/new.html.erb

```diff
@@ -14,11 +14,8 @@
   </div>
 
   <div class="field">
-    <%= f.fields_for :department do |d| %>
-      <%= d.label :department %>
-      <%= d.select :name,
-          Department::DEPARTMENT, class: "department dropdown"%>
-    <% end %>
+      <%= f.label :department_id %>
+      <%= collection_select(:department_id, :department_id, Department.all, :id, :name) %>
```

Comment:
you can definitely dissect the hash on the server, but it doesn't make much sense to me to have an object, because it's not a separate entity, right?  there's a field on the user table for department_id?  Or at least that's how it should be, otherwise you'll have to create a new department for each user.